hooks/jest.config.js

```diff
@@ -5,4 +5,7 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
+  moduleNameMapper: {
+    "^@/(.*)$": "<rootDir>/src/$1"
```

Comment:
How does this work? Because there is only one path with `src` in the old config (`"<rootDir>/notification/src"`). The others are w/o `src`.